internal/ctxwatch/context_watcher_test.go

```diff
@@ -87,6 +87,9 @@ func TestContextWatcherStress(t *testing.T) {
 		if i%2 == 1 {
 			cancel()
 		}
+
+		// To avoid context leak
+		cancel()
```

Comment:
I don't think there is a context leak here. It is always canceled in one of the if statements. If i is even it is canceled before Unwatch and if it is odd it is canceled after the Unwatch.

Reply:
Just linter notifies about it, so I decided to make it more clear.
Should I revert this?

Reply:
Yeah, I think so. It's annoying to have a linter warning, but I think that the logic of the test where half are canceled before and half canceled after is obscured by the superfluous cancel.

Reply:
Done)